src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -175,6 +175,9 @@ typedef struct wrapped_rr_closure_arg {
   /* The RR instance related to the closure */
   grpc_lb_policy* rr_policy;
 
+  /* The grpclb instance that created the wrapping */
```

Comment:
Might be a good idea to note that we're not holding a ref to the LB policy; the field is just here to provide the pointer value for log messages.

Reply:
Done